lms/djangoapps/mobile_api/groups/tests.py

```diff
@@ -17,32 +20,112 @@ def setUp(self):
         self.assertEqual(response.status_code, 200)
-        self.assertTrue('12345' in response.data['group-id'])  # pylint: disable=E1103
+        self.assertTrue('success' in response.data)  # pylint: disable=E1103
+
+        # Invite three members, two who are not in the group and one that is a member already.
+        member_ids = '366785273488903,939400156088941,10154831816670300'
+        url = reverse('invite-to-group', kwargs={'group_id' : group_id}) 
+        response = self.client.post(url, {  'member-ids' : member_ids })
+        self.assertEqual(response.status_code, 400)        
+
+        # Remove the members added
+        for member_id in member_ids.split(','):
+            url = reverse('group-remove-member', kwargs={ 'group_id' : group_id, 
+                                                      'member_id' : member_id}) 
+            response = self.client.delete(url)
 
-    def test_invite_members(self):
-        url = reverse('invite-to-group', kwargs={'group_id':'123456789'}) 
-        response = self.client.post(url, {  'member-ids': '1,2,3,4,5,6',
-                                            'oauth-token' : 'abcd1234'})
+    def test_delete_group(self): 
+        # Create new group
+        url = reverse('create-new-group')
+        response = self.client.post(url, {  'name' : 'TheBestGroup',
+                                            'description' : 'The group for the best people',
+                                            'privacy' : 'open'})
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('id' in response.data)  # pylint: disable=E1103
+        
+        url = reverse('delete-group', kwargs={'group_id' : response.data['id']}) 
+        response = self.client.delete(url)
         self.assertEqual(response.status_code, 200)
-        self.assertTrue('true' in response.data['success'])  # pylint: disable=E1103
 
-    def test_get_all_group_members(self):
-        # set_trace() 
-        url = reverse('members-in-group', kwargs={'group_id':'123456789'})
-        response = self.client.get(url)
+    def test_remove_member(self): 
+        group_id = '756869167741019'
+        member_id = '10154831816670300'
+        # Note that the memeber must be a member of the app and not a member 
+        # of the group in order to be able to join the group.
+        url = reverse('invite-to-group', kwargs={'group_id' : group_id}) 
+        response = self.client.post(url, {  'member-ids' : member_id })
         self.assertEqual(response.status_code, 200)
-        self.assertTrue('members' in response.data)  # pylint: disable=E1103
\ No newline at end of file
+        self.assertTrue('success' in response.data)  # pylint: disable=E1103
+        
+        # Remove member 
+        url = reverse('group-remove-member', kwargs={ 'group_id' : group_id, 
+                                                      'member_id' : member_id}) 
+        response = self.client.delete(url)
+
```

Comment:
Another general note --- if you find yourself duplicating code in test suites --- it's a good investment to create helper functions (as you would in prod. code)